custom_components/luxtronik/sensor.py

```diff
@@ -300,6 +301,19 @@ async def async_setup_entry(
             unit_of_measurement=ENERGY_KILO_WATT_HOUR,
             entity_category=EntityCategory.DIAGNOSTIC,
         ),
+        LuxtronikSensor(
```

Comment:
I prefer to check if the sensor is present. My value is always "None".
e.g.:
has_heat_output = luxtronik.get_value("calculations.Heat_Output") is not None
if has_heat_output:

Reply:
sorry, just copied the pattern from other sensors. Where should I place the "if"?

Reply:
Think I did it :-)